charts/kueue/templates/manager/manager.yaml

```diff
@@ -23,6 +23,9 @@ spec:
       - args:
         - --config=/controller_manager_config.yaml
         - --zap-log-level=2
+        {{- range .Values.controllerManager.featureGates }}
+        - --feature-gates={{ .name }}={{ .value }}
```

Comment:
Also verify whether feature gate will be overrided?

Reply:
Ad discussed offline, let's use 
```
controllerManager:
  featureGates: "a=b,c=d"
```

This is enough I think.

Reply:
done